janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/JanusGraphManagerServiceImpl.java

```diff
@@ -63,7 +66,31 @@ public void getJanusGraphContextByGraphName(GetJanusGraphContextByGraphNameReque
             responseObserver.onError(Status.INTERNAL
                 .withCause(new IllegalStateException("graphName should access a JanusGraph instance")).asException());
         }
-        responseObserver.onNext(JanusGraphContext.newBuilder().setGraphName(graphName).build());
+        responseObserver.onNext(GetJanusGraphContextByGraphNameResponse.newBuilder()
+            .setContext(JanusGraphContext.newBuilder().setGraphName(graphName))
+            .build());
+        responseObserver.onCompleted();
+    }
+
+    @Override
+    public void getVersion(GetVersionRequest request, StreamObserver<GetVersionResponse> responseObserver) {
+        if (request == null) {
+            responseObserver.onError(Status.INVALID_ARGUMENT
+                .withCause(new NullArgumentException("request should be set")).asRuntimeException());
+            return;
+        }
+        String tinkerPopVersion = "debug-tp";
+        String janusGraphVersion = "debug-jg";
```

Comment:
My plan was to leave it. The problem is if you run tests with IntelliJ the values are not set in the jar.

Reply:
I see. I don't know a better way to deal with it. That being said, can we use "Manifests.exists(name)" to check if they exist, rather than using try-catch block?